pages/urls.py

```
@@ -1,8 +1,11 @@
 from django.urls import path
 
-from .views import HomePageView, AboutPageView
+from .views import HomePageView, AboutPageView, syscall, ConnectAndRunScriptView
```

Pylint (3.3.7)

[error] No name 'syscall' in module 'pages.views'

(E0611)